packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Application} from 'probot';
+import {GitHubAPI} from 'probot/lib/github';
+import * as yaml from 'js-yaml';
+import * as util from 'util';
+
+const CONFIGURATION_FILE_PATH = '.bots/blunderbuss.yml';
+
+// const CONFIGURATION_FILE_PATH = '.kvgbot.yml';
+
+interface Configuration {
+  assign_issues?: string[];
+  assign_prs?: string[];
+}
+
+interface Repository {
+  owner: string;
+  repo: string;
+}
+
+
+// Fetches the config for the repo where the event occurred.
+async function fetchConfig(repo: Repository, github: GitHubAPI): Promise<Configuration> {
+  const response = await github.repos.getContents({
+    owner: repo.owner,
+    repo: repo.repo,
+    path: CONFIGURATION_FILE_PATH,
+  });
+  const fileContents = Buffer.from(
+      response.data.content,
+      'base64'
+  ).toString('utf8');
+  return yaml.safeLoad(fileContents) as Configuration;
+}
+
+// Returns a random item from an array
+function randomFrom(items: string[]): string {
+  return items[Math.floor(Math.random() * items.length)]
+}
+
+export = (app: Application) => {
+  app.on(['issues.opened', 'issues.reopened'], async context => {
+    const repo = context.repo() as Repository;
+    const config = await fetchConfig(repo, context.github);
+
+    if (!config.assign_issues) {
+      context.log.info(util.format('[%s/%s] issue %s ignored: not configured', repo.owner, repo.repo, context.payload.issue.id));
+      return;
+    }
+
+    if (context.payload.issue.assignees.length !== 0) {
+      context.log.info(util.format('[%s/%s] issue %s ignored: already has assignee(s)', repo.owner, repo.repo, context.payload.issue.id));
+      return;
+    }
+
+    const assignee = randomFrom(config.assign_issues);
```

Comment:
So if I'm reading this right - we would need to configure a config file in each repo to find who the potential assignees are?

Reply:
I was wondering about this, how does this work today? I wonder if we could come up with some heuristics for loading lists of folks from, as an example, a specific team on GitHub?

For testing a configuration file seems reasonable.